### ISSUE TITLE
Handling angleToleranceInterval for tangible device as special case

### ISSUE THREAD

**Issue by brettfiedler** [phetsims/quadrilateral#47]
Tangible input lacks a "pressed" or "focused" property. This means it will not properly interact with the way we implemented the scaleFactors for touch/cursor/keyboard input.

Let's consider the best way to set the appropriate tolerance values for the case where a tangible device in connected.

**Comment by jessegreenberg** [comment]
Here is how it works for mouse/touch:

```js
        if ( anySidesPressed && this.isParallelogramProperty.value ) {

          // A side has been picked up while the shape is a parallelogram - it should be impossible for the shape
          // to go "out" of parallelogram in this case because none of the angles should be changing (true if we were
          // a perfect parallelogram)
          toleranceInterval = Number.POSITIVE_INFINITY;
        }
        else if ( numberOfVerticesPressed >= 2 ) {

          // Two or more vertices pressed at once, increase the tolerance interval by a scale factor so that
          // it is easier to find and remain a parallelogram with this input
          toleranceInterval = QuadrilateralQueryParameters.angleToleranceInterval * QuadrilateralQueryParameters.angleToleranceIntervalScaleFactor;
        }
        else if ( numberOfVerticesPressed === 1 ) {

          // Only one vertex is moving, we can afford to be as precise as possible from this form of input, and
          // so we have the smallest tolerance interval.
          toleranceInterval = QuadrilateralQueryParameters.angleToleranceInterval;
        }
        else {

          // We are dragging a side while out of parallelogram, or we just released all sides and vertices. Do NOT
          // change the angleToleranceInterval because we don't want the quadrilateral to suddenly appear out of
          // parallelogram at the end of the interaction. The ternary handles initialization.
          toleranceInterval = this.angleToleranceIntervalProperty ? this.angleToleranceIntervalProperty.value : QuadrilateralQueryParameters.angleToleranceInterval;
        }
```

**Comment by jessegreenberg** [comment]
The implementation of the tangible control is sort of like dragging all four vertices at once all the time. Each frame a new position for the vertex is given to the sim from the physical sensors.

This is kind of like dragging multiple vertices at once. When the physical device is connected we could just a larger tolerance, like 
```js
toleranceInterval = QuadrilateralQueryParameters.angleToleranceInterval * QuadrilateralQueryParameters.angleToleranceIntervalScaleFactor;
```
or more.

**Comment by BLFiedler** [comment]
If we're using some variable to detect if a device is connected, we need to smoothly allow input switching to mouse/touch/keyboard even if the device is still plugged in (just not being used).

- Mouse/Keyboard/Touch should take priority over tangible input, but when non-tangible input is not active and the sim receives updated tangible values, the sim should realign with tangible values. 
If a device is behaving properly, we only need to check for new values.

- JG suggests that using the 'pressed' status of mouse/touch/keyboard could prevent tangible input while moving a side or vertex using mouse/touch/keyboard. Implication being if a device is misbehaving and sending updated values it will revert to the behavior in the previous bullet point.

**Comment by BLFiedler** [comment]
During design meeting we decided to create a query parameter for the case of a tangible device connected to the sim that will make the angleToleranceInterval always = angleToleranceInterval*scaleFactor (for now).

**Comment by jessegreenberg** [comment]
Done in the above commits. Outside of the electron app, the angleToleranceIntervalProperty behaves like https://github.com/phetsims/quadrilateral/issues/47#issuecomment-1017857743. When connected to the app (detected with query param `?deviceConnection`, which is already used to include the "callibration" dialog) the angleToleranceIntervalProperty is always 
```js
QuadrilateralQueryParameters.angleToleranceInterval * QuadrilateralQueryParameters.angleToleranceIntervalScaleFactor;
```

I rebuilt the app in windows and macOS with this change. @BLFiedler is there anything else for this issue?
